src/app/base/components/Header/Header.test.tsx

```diff
@@ -1,341 +1,266 @@
-import { screen, within } from "@testing-library/react";
```

Comment:
Only the last 2 tests are new in this file, the rest are rewrites with the store included